src/translations/en.json

```diff
@@ -269,18 +269,26 @@
       "refresh": "Refresh",
       "cancel": "Cancel",
       "delete": "Delete",
+      "remove": "Remove",
+      "enable": "Enable",
+      "disable": "Disable",
       "close": "Close",
+      "leave": "Leave",
+      "stay": "Stay",
       "next": "Next",
+      "back": "Back",
       "undo": "Undo",
       "save": "Save",
+      "rename": "Rename",
```

Comment:
This one is used once, not really common?

Reply:
I put it in here, since I interpreted "common" to also also mean "generic" => it doesn't make sense to stick this really generic simple string into a specific subsection IMHO.

I wouldn't be surprised if other processes in the future would require the same as well.